.github/actions/pnpm/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Setup node and pnpm
+description: Setup node and install dependencies using pnpm
+runs:
+  using: "composite"
+  steps:
+    - uses: volta-cli/action@v1
+    - name: Cache pnpm modules
+      uses: actions/cache@v3
+      with:
+        path: ~/.pnpm-store
```

Comment:
is this because we aren't using setup node?

Reply:
It isn't. The pnpm store has its own location. Separate from npm stuff.

And i use volta, so i never need to manually setup node.

Reply:
I don't think you understand, but you did answer the question and the answer is "yes". Btw setupNode action has built in cache restore for pnpm yarn and npm. It's not just for npm. I too use Volta but not for CI since that slows actions down quite a bit.

Reply:
oh, I didn't know that. neat.

how do we make volta faster?
does it need a cache?